scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -469,7 +469,7 @@ task M2 {
         --max-mnp-distance 0
   >>>
   runtime {
-      docker: select_first([gatk_docker_override, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
```

Comment:
@aofarrel I think we should keep directing to the gcr repo, but use the newer gatk.

Reply:
Is there a meaningful difference between gcr vs dockerhub in this context? More people are familiar with the latter so that is what I defaulted to.

Reply:
Fixed in the newest commit.